Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -260,6 +260,11 @@ theorem rTensor_preserves_injective_linearMap {N' : Type*} [AddCommGroup N'] [Mo
 @[deprecated (since := "2024-03-29")]
 alias preserves_injective_linearMap := rTensor_preserves_injective_linearMap
 
+instance [Flat R M] [Flat R N] : Flat R (M ⊗[R] N) :=
+  (iff_rTensor_injective' _ _).mpr fun I ↦ by
+    simpa [rTensor_tensor] using rTensor_preserves_injective_linearMap _
+      (rTensor_preserves_injective_linearMap _ I.injective_subtype)
```

Comment:
Is it easy to get the heterobase version (i.e. `[Flat S M] [Flat R N] : Flat S (M ⊗[R] N)`)

Reply:
Done!